packages/devtools_app/lib/src/flutter/theme.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+ThemeData themeFor({@required bool isDarkTheme}) {
+  final theme = isDarkTheme ? _darkTheme() : _lightTheme();
+  return theme;
+}
+
+ThemeData _darkTheme() {
+  final theme = ThemeData.dark();
+  return theme.copyWith(
+      primaryColor: devtoolsGrey[900],
+      primaryColorDark: devtoolsBlue[700],
+      primaryColorLight: devtoolsBlue[400],
+      indicatorColor: devtoolsBlue[400],
+      accentColor: devtoolsBlue[400],
+      backgroundColor: devtoolsGrey[600],
+      buttonTheme: theme.buttonTheme.copyWith(minWidth: buttonMinWidth));
+}
+
+ThemeData _lightTheme() {
+  final theme = ThemeData.light();
+  return theme.copyWith(
+      primaryColor: devtoolsBlue[600],
+      primaryColorDark: devtoolsBlue[700],
+      primaryColorLight: devtoolsBlue[400],
+      indicatorColor: Colors.yellowAccent[400],
```

Comment:
fwiw, the existing yellow accent in devtools is pretty ugly. I'm completely open to a better or more standard accent color.

Reply:
The yellow in the inspector?  I can agree to that.  You can check the screenshot above;  this is a very different shade of yellow.

For alternatives, we can get a more subdued effect with the subdued branded grey100, or the branded blue400.

As a Seattlite, I'm obligated to recommend a healthy neon green.  limeAccent400 is great on blues.